src/test/java/bio/terra/common/logging/LoggingTest.java

```diff
@@ -201,8 +196,8 @@ public void testStructuredLogging() {
             .read(path);
   }
 
-  private String lastLoggedLine() {
-    String[] lines = outputCapture.getAll().split("\n");
+  private String lastLoggedLine(CapturedOutput capturedOutput) {
```

Comment:
should probably be Optional<String> in case there are no lines in the capture.

Reply:
That might be a tad cleaner, but IMO it's good enough for the test to fail on an NPE here. If you're asking for the last logged line and none exists, the case should fail hard & fast.